src/components/SearchList.js

```diff
@@ -30,9 +29,9 @@ const SearchList = ({sites, getSiteDetail, navigate}) => {
     } else {
         return (
             <ScrollView>
-                <List>
+                <View>
                     {renderCampsiteListItemList({sites, getSiteDetail, navigate})}
-                </List>
```

Comment:
`List` was deprecated in the RNE upgrade, so we had to replace it everywhere we used it.